File: LatA/cpu.cpp
#include <iostream>
#include <string>
using namespace std;
class cpu{
    // Attribute
    private:
        string nama;
        int harga;
    public:
        //Constructor kosong
        cpu(){}
        // Constructor
        cpu(string name, int price){
            this->nama = name;
            this->harga = price;
        }
        // get dan Set
        void setNama(string name){
            this->nama = name;
        }
        string getNama(){
            return this->nama;
        }
        void setHarga(int price){
            this->harga = price;
        }
        int getHarga(){
            return this->harga;
        }
        // Fungsi untuk menampilkan isi attribute cpu
        void printCpu(){
            cout << "CPU Name: " << this->nama << endl;
            cout << "CPU Price: " << this->harga << endl;
        }
        // Destructor
        ~cpu(){}
};

File: LatB/Php/Cpu.php
<?php
class Cpu{
    #attribute untuk class CPU
    private $nama;
    private $harga;
    function __construct($nama, $harga){
        $this->nama = $nama;
        $this->harga = $harga;

    }
    # Get dan Set untuk semua attribute Private
    function setNama($nama){
        $this->nama = $nama;
    }
    function getNama(){
        return $this->nama;
    }
    function setHarga($harga){
        $this->harga = $harga;
    }
    function getHarga(){
        return $this->harga;
    }
    #Fungsi untuk menampilkan isi dari attribute private di class CPU
    function printCpu(){
        echo "CPU Name: " .$this->nama."<br>";
        echo "CPU Price: " .$this->harga."<br>";
    }
    function __destruct()
    {
        
    }
}
?>